modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/node/test/TaskNodeExecutorTest.java

```diff
@@ -271,19 +270,31 @@ private void _executeTimer(
 			KaleoTimerInstanceToken kaleoTimerInstanceToken)
 		throws Exception {
 
-		Method executeTimerMethod = ReflectionUtil.getDeclaredMethod(
-			_nodeExecutor.getClass(), "executeTimer", ExecutionContext.class);
+		Bundle bundle = FrameworkUtil.getBundle(_nodeExecutor.getClass());
 
-		executeTimerMethod.invoke(
-			_nodeExecutor,
-			new ExecutionContext(
-				kaleoInstanceToken, kaleoTimerInstanceToken, _workflowContext,
-				_serviceContext) {
+		BundleContext bundleContext = bundle.getBundleContext();
 
-				{
-					setKaleoTaskInstanceToken(kaleoTaskInstanceToken);
-				}
-			});
+		ServiceReference<?> serviceReference =
```

Comment:
HEy @regiCesar21, create a field for this and using the `@Inject` with a filter the get the `TimerExecutor` instance.